src/App.vue

```diff
@@ -424,6 +487,197 @@ export default {
     ]),
     ...mapActions("selectors", ["hideSelector"]),
     ...Utilities,
+    /**
+     * NOTE - In this method, "org" will be used in place of "workspace"
+     * to keep variable names short and readable.
```

Comment:
So this is what we're deciding on now right? org -> workspace everywhere on the frontend?
I'll make this change plus all other too. Otherwise in next PR someone will have to do these changes in rest of the files.

I think we should write these things down so we have everything agreed upon the next time there's some difference in opinions. I've made a list in Engineering Wiki

https://www.notion.so/avantifellows/Misc-PR-review-agreements-431435cc040e4dbe9a127e8124973d75
Kuch cheez ke saath disagree krta hai toh udhar hi update krde